### PR TITLE
nfv templates 

### PR DESCRIPTION
**Pull request by pinikomarov**
This commit replaces pr416 (has less commits )

### CODE REVIEW

**Comment by pinikomarov** [comment]
@olliewalsh @abays can you give this an "ok to test" ? so it can test the rebase D/S ?

File: ansible/vars/local_templates_hybrid_dpdk_sriov.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+
+#change this to local-defaults.yaml to have affect 
+---
+osp_extra_env_files: ["services/neutron-ovs-dpdk.yaml"]
+
+osp_extra_env_files: ["services/neutron-ovs.yaml","services/neutron-ovs-dpdk.yaml","services/neutron-sriov.yaml"]
```
Comment (abays) [inline ansible/vars/local_templates_hybrid_dpdk_sriov.yaml:6]: Looks like line 4 can just be deleted, as its contents are included on line 6

Comment (pinikomarov) [reply ansible/vars/local_templates_hybrid_dpdk_sriov.yaml]: oh right ,ok removed that

**Comment by pinikomarov** [comment]
@abays @olliewalsh Hi , so are we good here ?

**Review by abays — COMMENTED** [review]
+1 from me

**Pull request merged**